Cargo.toml

```diff
@@ -54,6 +54,7 @@ rustyline-derive = { version = "0.6.0", optional = true }
 
 # The `wasm-bindgen` version is pinned (`=`) because it must be the same as `wasm-bindgen-cli` in `flake.nix`
 wasm-bindgen = { version = "=0.2.83", optional = true, features = ["serde-serialize"] }
+serde-wasm-bindgen = "0.4.5"
```

Comment:
This new dependency was necessary to replace some deprecated code, see e.g. `src/repl/wasm_frontend.rs`